### ISSUE TITLE
Use correctly shuffle=False in fit_generator() function

### ISSUE THREAD

**Issue by victor-robotic-24** [keras-team/keras#14275]
Shuffle parameter in fit_generator function can take two boolean values: True or False. I want to execute fit_generator without altering the order of the batches at each epoch of the training, so I have to assign 'shuffle=False'. Also, I want to specify the number of steps per epoch, but shuffle parameter has effect when 'steps_per_epoch=None'. 
- Is there any way to solve that? 
- How can I use correctly shuffle parameter in fit_generator function? 
- Is there any way to check batches at each epoch are not being shuffling?

**Comment by PfeifferJosef-96** [comment]
Hello @victor-robotic-24 ,

could you please provide the keras version you are using?

The fit_generator method is already deprecated in currently newest version. You could instead use the fit()-method, as it also accepts a generator as input / validation.

Also - as stated in current [documentation](https://keras.io/api/models/model_training_apis/#fit-method):
- Shuffle argument is ignored if input (x) is a generator
- Shuffle argument Has **no** effect when steps_per_epoch is not None. -> setting steps_per_epoch results that "shuffle" has no  effect.

To have full control over what is happening, try to use [keras.utils.Sequence](https://keras.io/api/utils/python_utils/#sequence-class) as generator.

**Comment by victor-robotic-24** [comment]
I am using Keras 2.2.4. Is it deprecated fit_generator for this version of Keras? Would be more correct use fit()?

**Comment by sushreebarsa** [comment]
@victor-robotic-24 Moving this issue to closed status as there has been no recent activity.In case you still face the error please create a new issue,we will get you the right help.Thanks!
